README.md

```diff
@@ -979,7 +979,9 @@ Valid keys and their meanings:
 
 * **write_to_cache** - Accepts values `true` or `false`.  If `false`, the completed calls from this workflow will not be added to the cache.  See the [Call Caching](#call-caching) section for more details.
 * **read_from_cache** - Accepts values `true` or `false`.  If `false`, Cromwell will not search the cache when invoking a call (i.e. every call will be executed unconditionally).  See the [Call Caching](#call-caching) section for more details.
+* **workflow_log_dir** - Specifies a path where per-workflow logs will be written.  If this is not specified, per-workflow logs will not be generated.
 * **outputs_path** - Specifies a path where final workflow outputs will be written.  If this is not specified, workflow outputs will not be copied out of the Cromwell workflow execution directory/path.
+* **call_logs_dir** - Specifies a path where final call logs will be written.  If this is not specified, call logs will not be copied out of the Cromwell workflow execution directory/path.
```

Comment:
Yes, that _should_ work... but we don't have automated tests. Let me get back to you? It'll take me a bit to re-setup JES on my machine.

I just refactored and used the existing "copy workflow outputs via `Files.copy`" to a "copy any resolved `Path` via `Files.copy`", internally invoking the existing `GcsFileSystemProvider`.